test/e2e/kubernetes/storageclass/storageclass.go

```diff
@@ -77,6 +96,20 @@ func (d *Builder)WithWaitForFirstConsumer() *Builder  {
 	return d
 }
 
+// WithOotProvisionerName sets the correct external azure disk csi driver provisoner name
+func (d *Builder)WithOotProvisionerName() *Builder  {
```

Comment:
```suggestion
func (d *Builder) WithOotProvisionerName() *Builder  {
```
Not sure why the existing functions didn't have a space but I think we can go back and add it.